package.json

```diff
@@ -223,6 +233,16 @@
         "command": "extension.vim_ctrl+p",
         "when": "suggestWidgetVisible && vim.active && vim.use<C-p>"
       },
+      {
+        "key": "ctrl+p",
+        "command": "extension.vim_ctrl+p",
+        "when": "vim.mode == 'SearchInProgressMode' && vim.active && vim.use<C-p>"
```

Comment:
I think it was a mistake to not have !inDebugRepl in the previous one

Reply:
Should I add it to the existing ctrl+p binding?

Reply:
I'm assuming the answer to that is "yes". Resolved.

Reply:
thanks! Sorry to make you assume :)